DESCRIPTION

```diff
@@ -2,24 +2,20 @@ Package: shiny.router
 Type: Package
 Title: Basic Routing for Shiny Web Applications
 Version: 0.2.3
-Authors@R: c(person("Filip", "Stachura",
-                    email = "filip@appsilon.com",
-                    role = c("aut")),
-            person("Dominik", "Krzemiński",
-                    email = "dominik@appsilon.com",
-                    role = c("aut")),
-            person("Krystian", "Igras",
-                    email = "krystian@appsilon.com",
-                    role = c("aut")),
-            person("Developers", "Appsilon",
-                    email = "support+opensource@appsilon.com",
-                    role = c("cre", "aut")),
-              person("Servet Ahmet", "Çizmeli", email = "ahmet.cizmeli@contractors.appsilon.com", role = c("ctb")),
-            person(family = "Appsilon", role = c("cph"))
-           )
+Authors@R:
+  c(
+    person("Ryszard", "Szymański", email = "opensource+ryszard@appsilon.com", role = c("cre", "aut")),
```

Comment:
opensource+ryszard@appsilon.com - does an email like that exist or did you mean `ryszard@appsilon.com`?

Reply:
`+` in Gmail is used to add something to the email address, so in fact, this will work the same as `opensource@appslion.com`, but it will be easy to filter messages. CRAN policy requires the maintainer to be a real person, not an organization (as previously), but we also want to be able to access CRAN communication in a case when the maintainer is not available. Now I will be able to e.g. confirm a new release without using your precious time ;) 
I also recommend requesting access to `opensource@appsilon.com`.